HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Unreleased Changes
 ------------------
+* The Finfish Aquaculture model know longer generates histograms for
+  uncertainity analysis due to issues with matplotlib that make invest
+  unstable. See https://github.com/natcap/invest/issues/87 for more.
```

Comment:
Sorry, this is pretty trivial, but there are a couple typos: `know` --> `no` and `invest` --> `InVEST`.

Reply:
Thanks, that was pretty sloppy.